bundler/lib/bundler/templates/newgem/.gitlab-ci.yml.tt

```diff
@@ -0,0 +1,9 @@
+image: ruby:<%= RUBY_VERSION %>
+
+before_script:
+  - gem install bundler
+  - bundle install -v <%= Bundler::VERSION %>
```

Comment:
This command is incorrect, the `-v <%= Bundler::VERSION %>` should go in the line above.